### DISCUSSION TITLE
Using the grid only

### DISCUSSION THREAD

**Discussion by antangible** [tailwindlabs/tailwindcss#6872] [Help]
Hi, is there a way to import just the grid without anything else?

**Comment by ProPhycient5** [comment]
@antangible import `grid` means ???

**Comment by adamwathan** [comment]
You can use the `corePlugins` configuration to disable every feature of Tailwind except the grid features:

https://tailwindcss.com/docs/configuration#core-plugins

Tailwind only generates the classss you actually use though so if you just don’t use anything from Tailwind except grid related classes that will work fine too.
